src/test/kotlin/FollowerTest.kt

```diff
@@ -11,16 +13,16 @@ import org.junit.jupiter.api.Test
 class FollowerTest {
     @Test
     fun `Do not vote for more than one Candidate`() {
-        val network = Network(mapOf(Address("host", 1) to listOf(
-            NetworkMessage(RequestForVotes(Address("host", 2), Address("host", 1), "REQUEST FOR VOTES"), 0),
-            NetworkMessage(RequestForVotes(Address("host", 3), Address("host", 1), "REQUEST FOR VOTES"), 0)
+        val network = Network(mapOf(Source("host", 1) to listOf(
+            NetworkMessage(RequestForVotes(Source("host", 2), Destination("host", 1), "REQUEST FOR VOTES"), 0),
+            NetworkMessage(RequestForVotes(Source("host", 3), Destination("host", 1), "REQUEST FOR VOTES"), 0)
         )))
 
         //TODO Write an ADR on why use time machine instead of tick on the Node and Network and why the Network is mutable
-        val timeMachine = TimeMachine(network, Follower(Address("host", 1), "follower", 0, network, emptyList(), emptyList()))
+        val timeMachine = TimeMachine(network, Follower(Source("host", 1), "follower", 0, network, emptyList(), emptyList()))
         val (_, follower) = timeMachine.tick()
 
         //TODO COULD BE THAT THE ADDRESSES ARE WRONG
```

Comment:
We should probably remove this comment 🤔

Reply:
Great catch